packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.2.0]
+
+### Added
+
+- `AccountsController:accountAdded` and `AccountsController:accountRemoved` events added. ([#4496](https://github.com/MetaMask/core/pull/4496))
+- Add and export corresponding event types `AccountsControllerAccountAddedEvent`, `AccountsControllerAccountRemovedEvent`.
+- Export action types `AccountsControllerListMultichainAccounts`,`AccountsControllerGetSelectedMultichainAccount`,`AccountsControllerGetNextAvailableAccountName}Action`. ([#4497](https://github.com/MetaMask/core/pull/4497))
+
+### Fixed
+
+- Use `listMultichainAccounts` instead of `listAccounts` for non-EVM specific multichain methods. ([#4494](https://github.com/MetaMask/core/pull/4494))
+- For initial account, emit `selectedAccountChange ` and update `lastSelected`. ([#4494](https://github.com/MetaMask/core/pull/4494))
```

Comment:
RE https://github.com/MetaMask/core/pull/4498#discussion_r1664533254, if these are not user-facing changes or implementation details, should they be removed from the changelog? Or maybe we could add a comment about how the bug affected controller behavior and how that was changed?

```suggestion
```

Reply:
It seems like both of these PR have consumer-facing changes, so it makes sense to record changelog entries for them, but it's unclear to me what those changes are based on looking at this PR.

The original ticket tied to this PR seems to indicate that previously, the Snap account number was not incremented correctly. I presume that this PR fixed that bug, so perhaps we should make a reference to that bug here? I was not able to locate a test for the desired behavior, so I am also unsure how to best represent that here.

If both of these change descriptions make perfect sense to your team without needing to dive into the PR or original ticket, it's fine to leave both of these entries as they are, but you may consider rewording them in such a way that it's more obvious what got fixed.

Reply:
Alright, then I might consider rewording those and probably move them to a "CHANGED" section, since those are also new changes we needed to fully support non-EVM in various contexts.

This also fixed a bug with account numbers yes, but ultimately we needed those changes anyway! :)

I'll try to change that so that we can validate this release.